recipes/hdfs/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
+if errorlevel 1 exit 1
+
+:: Add more build steps here, if they are necessary.
+
+:: See
+:: http://docs.continuum.io/conda/build.html
+:: for a list of environment variables that are set during the build process.
```

Comment:
When you have single line install commands you can delete the `build.sh` and the `bld.bat` and add a `script` field in the `build` section inside your `meta.yaml` file. (See below.)